### ISSUE TITLE
Batch scripts and manual device binding for XSEDE

### ISSUE THREAD

**Issue by sbryngelson** [MFlowCode/MFC-develop#23]
We need a version of the code pushed that makes use of non-Summit parallel resources on GPU. This is likely a combination of some device bindings in p_main and some modifications of python proxy.

**Comment by sbryngelson** [comment]
@mrodrig6: @anandrdbz needs these for working on xsede and local compute resources.

**Comment by sbryngelson** [comment]
@mrodrig6 still waiting on this one...

**Comment by mrodrig6** [comment]
@sbryngelson this was pushed into the GPU-production branch (now defunct), was this found? If not, I can add it to the GPU-3D-managed branch, thoughts?

**Comment by mrodrig6** [comment]
@sbryngelson Manual binding has been added to the phasechange branch, I tried to push the GPU-3D-unmanaged and it appears only pulls are allowed. On phasechange branch, the bindings do not compile because the preprocessor directive fails, will ask Henry about this.

**Comment by sbryngelson** [comment]
@henryleberre can help. perhaps merge in. just need to be sure this doesn't mess up runs on summit like systems that use jsrun.

**Comment by mrodrig6** [comment]
@sbryngelson @henryleberre has helped me resolved the issue and phasechange has the bindings. Once we start the merger (tomorrow morning) between phase & GPU-3D we can make sure to add the bindings.

**Comment by sbryngelson** [comment]
@henryleberre @anandrdbz I've moved this issue to you guys. I know the bindings work on non-Summit-like machines (e.g., XSEDE), but we should
1. Be sure these bindings have been added to the GPU branch
2. Be sure they work as expected on Summit

I believe these bindings assume 1 MPI rank per GPU, which seems fine. I will open a separate issue to throw an error (via `m_python_proxy.py`) if someone attempts to use more MPI ranks than GPUs.
